### DISCUSSION TITLE
Super minor, add markdown emoji support to article titles

### DISCUSSION THREAD

**Discussion by jcla490** [jpanther/congo#89] [Ideas]
I noted that titles weren't capable of displaying emojis when using the syntax like `:grapes:` and instead required me to copy and paste. 

In `article-link.html`, I've added `{{ .Title | emojify}}` instead of `{{ .Title}}` and this took care of the issue. 

Very minor... would be cool to have it around in v2, I use emojis to categorize my spirits reviews (i.e. `title: "🇫🇷 🍇 Armagnac | Grosperrin Lot 1996"`)

**Comment by jpanther** [comment]
Hey @jcla490, this was actually fixed in v1.6.3. Are you using the latest version?? See https://github.com/jpanther/congo/pull/84.

**Reply by jcla490** [reply]
oof, sorry! I'll update now, much thanks for all your work making congo!
